semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -247,7 +247,48 @@ def __init__(self, group):
 class MetricSemanticConvention(BaseSemanticConvention):
     GROUP_TYPE_NAME = "metric"
 
-    allowed_keys = ()
+    allowed_keys: Tuple[str, ...] = BaseSemanticConvention.allowed_keys + (
+        "metric_name",
+        "units",
+        "instrument",
+    )
+
+    yaml_to_markdown_instrument_repr: Dict[str, str] = {
+        "counter": "Counter",
+        "updowncounter": "UpDownCounter",
+        "histogram": "Histogram",
```

Comment:
The exponential histogram isn't an instrument, rather it's a method of aggregation. Notice how the [API spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument) doesn't mention exponential histograms. This is because the exponential histogram is purely an SDK/OTLP concept, where users would specify how to aggregate/export measurements given by instruments. 

Since auto-instrumentation only has access to to API, this means that the semantic convention can't and shouldn't specify aggregation types, this is left up to whoever configures the SDK.

Reply:
True, I missed that. All good, you can resolve this :)